performance_regression_test.py

```diff
@@ -248,6 +259,7 @@ def run_workload(self, stress_cmd, nemesis=False):
             time.sleep(interval * 60)  # Sleeping one interval (in minutes) before starting the nemesis
             self.db_cluster.add_nemesis(nemesis=self.get_nemesis_class(), tester_obj=self)
             self.db_cluster.start_nemesis(interval=interval)
+            self._stop_load_after_one_nemesis_cycle()
         results = self.get_stress_results(queue=stress_queue)
         self.update_test_details(scrap_metrics_step=60)
```

Comment:
Did you check, that c-s results properly parsed and collected?
i suspect, that when you kill the c-s commands, the c-s tool doesn't write summary to the log and in this case it will not properly parsed or don't parsed at all for current tests.

Reply:
according to @fgelcer we don't need c-s tool summary in performance tests.

Reply:
@aleksbykov , IIRC in the latency with nemesis test, we don't do anything with this summary, do we?
maybe we just upload the results to ES, but it is not used to anything, as the latency values are calculated from Prometheus during each one of the operations..
maybe it can fail the final part, where we upload the data to ES, not sure about it...